ext/dom/node.c

```diff
@@ -1643,13 +1994,15 @@ PHP_METHOD(DOMNode, lookupPrefix)
 	char *uri;
 
 	id = ZEND_THIS;
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s", &uri, &uri_len) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), modern ? "s!" : "s", &uri, &uri_len) == FAILURE) {
```

Comment:
I forgot, but does `uri` get initialized to `NULL` if it is not past or not?

Reply:
It's not optional, so it must be passed. So it will always either be NULL if the user passed null, or a string if the user passed a string.

Reply:
Right, I read the ZPP change wrong